pom.xml

```diff
@@ -301,92 +85,19 @@
 					<useFile>true</useFile>
 				</configuration>
 			</plugin>
-            <!-- ikke oppgrader til 1.2.0 før weld støtter nytt indexformat -->
-			<plugin>
-				<groupId>org.jboss.jandex</groupId>
-				<artifactId>jandex-maven-plugin</artifactId>
-				<version>1.2.1</version>
-				<executions>
-					<execution>
-						<id>make-index</id>
-						<goals>
-							<goal>jandex</goal>
-						</goals>
-					</execution>
-				</executions>
-			</plugin>
 			<plugin>
 				<groupId>com.github.ferstl</groupId>
 				<artifactId>depgraph-maven-plugin</artifactId>
-				<version>4.0.1</version>
-				<configuration />
-
 			</plugin>
 		</plugins>
 	</build>
+
 	<distributionManagement>
 		<repository>
 			<id>github</id>
 			<name>Github navikt Maven Packages</name>
 			<url>https://maven.pkg.github.com/navikt/fp-felles</url>
 		</repository>
 	</distributionManagement>
-	<dependencies>
```

Comment:
Disse ble lagt inn for å unngå duplisering i lokale poms.
Ligger disse i fp-bom eller er den bare dependencyManagement ? Hvis ikke så ser du fort effekten når du bygger en app med en lokal snapshot-versjon av felles

Reply:
Stemmer - de er flyttet til bom'en nå. Så funksjonelt blir det det samme.